mysqld-exporter/main.libsonnet

```diff
@@ -9,6 +9,7 @@ local container = k.core.v1.container;
     user,
     host,
     port=3306,
+    tlsMode='preferred',
```

Comment:
Might be better to put this arg last in the list as some may be using unnamed arts and have the image name it this place.

Reply:
Fair point! I'll do that.